docs/format/source/format_release_notes.rst

```diff
@@ -196,23 +196,26 @@ Improved storage of ROIs
 * **Make links explicit:** The relationship of ``RoiResponseSeries`` to ``ROI`` objects was implicit (i.e. ROI was
   specified by a string), so one had to know a priori which ``ImageSegmentation`` and ``ImagingPlane`` was used
   to produce the ROIs.
+* **Support 3D ROIs:** Allow users to add 3D ROIs collected from a multi-plane image.
 
 **Changes:** See also `PR391 (PyNWB) <https://github.com/NeurodataWithoutBorders/pynwb/pull/391>`_ and
 `I118 (nwb-schema) <https://github.com/NeurodataWithoutBorders/nwb-schema/issues/118>`_ for details:
```

Comment:
Can you also add a link to https://github.com/NeurodataWithoutBorders/pynwb/pull/688 and https://github.com/NeurodataWithoutBorders/pynwb/issues/554 for the 3D ROI changes here.